src/devices/machine/acorn_memc.cpp

```diff
@@ -324,8 +324,15 @@ void acorn_memc_device::page_w(offs_t offset, uint32_t data, uint32_t mem_mask)
 	// always make sure ROM mode is disconnected when this occurs
 	m_latchrom = false;
 
+	phyaddr += memc * 0x80;
+
+	// unmap all logical pages that resolve to the same physical address
```

Comment:
Are you sure (ensuring one logical page per physical one) ?

No mmu I know of (I don't know acorn's) does that, because 1/ it's hard to do in hardware 2/ page sharing is useful and used a lot, even page sharing with different attributes.  I'm very surprised a mmu would exist which would enforce such a thing.